data/transportation/street-network-analysis/index.html

```diff
@@ -45,3 +51,21 @@ <h3 id="StreetNetwork">Street Network</h3>
     info=page.StreetNetwork
   %}
 </div>
+<div class="grid package">
+  <div class="grid__col grid__col--12-of-12">
+    <h3 id="StreetNetwork">Multimodal Network</h3>
```

Comment:
```suggestion
    <h3 id="MultiModalNetwork">Multimodal Network</h3>
```

Reply:
nice!  thx, yup, i just found that, too.  i appreciate it!